twisted/test/test_jelly.py

```diff
@@ -224,13 +232,18 @@ def test_typeBuiltin(self):
 
 
     def test_dateTime(self):
+        """
+        Jellying L{datetime.timedelta} instances and then unjellying the result
+        should produce objects which represent the values of the original
+        inputs.
+        """
         dtn = datetime.datetime.now()
         dtd = datetime.datetime.now() - dtn
-        input = [dtn, dtd]
-        c = jelly.jelly(input)
+        inputList = [dtn, dtd]
```

Comment:
this feels like Hungarian notation... https://en.wikipedia.org/wiki/Hungarian_notation

don't use it :)

what is wrong with `input` ?